pkg/handler/task_info.go

```diff
@@ -75,17 +82,28 @@ func (taskInfoHandler *TaskInfoHandler) listTasks(c echo.Context) error {
 // @Failure      404 {object} ce.ErrorResponse
 // @Failure      500 {object} ce.ErrorResponse
 // @Router       /tasks/{uuid} [get]
-func (taskInfoHandler *TaskInfoHandler) fetch(c echo.Context) error {
+func (t *TaskInfoHandler) fetch(c echo.Context) error {
 	_, orgID := getAccountIdOrgId(c)
 	id := c.Param("uuid")
 
-	response, err := taskInfoHandler.DaoRegistry.TaskInfo.Fetch(orgID, id)
+	response, err := t.DaoRegistry.TaskInfo.Fetch(orgID, id)
 	if err != nil {
 		return ce.NewErrorResponse(ce.HttpCodeForDaoError(err), "Error fetching task", err.Error())
 	}
 	return c.JSON(http.StatusOK, response)
 }
 
+// TODO remove, used for testing
+func (t *TaskInfoHandler) cancel(c echo.Context) error {
```

Comment:
IDK, we may consider keeping it

Reply:
It would be nice to have. I can leave it in if we want.